README.md

```diff
@@ -2,7 +2,9 @@
 
 A library for manipulating bioinformatics sequencing formats in Apache Spark.
 
-*NOTE: this is alpha software - everything is in flux at the moment*
+[![Build Status](https://travis-ci.org/disq-bio/disq.svg?branch=master)](https://travis-ci.org/disq-bio/disq)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.disq-bio/disq/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.disq-bio/disq)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Add javadoc badge
```
[![API Documentation](http://javadoc.io/badge/org.disq-bio/disq.svg?color=brightgreen&label=javadoc)](http://javadoc.io/doc/org.disq-bio/disq)
```

Reply:
Thanks for the suggestion @heuermh - I didn't know docs were automatically aggregated on that site.